pina/model/layers/convolution_2d.py

```diff
@@ -0,0 +1,292 @@
+"""Module for Continuous Convolution class."""
+
+import copy
+from convolution import BaseContinuousConv
+from utils import Integral, NeuralNet, create_stride, check_point, map_points_
+import torch
+
+
+class ContinuousConv2D(BaseContinuousConv):
+    """
+    Implementation of Continuous Convolutional operator.
+
+    .. seealso::
+
+    **Original reference**: Coscia, D., Meneghetti, L., Demo, N.,
+    Stabile, G., & Rozza, G.. (2022). A Continuous Convolutional Trainable
+    Filter for Modelling Unstructured Data.
+    DOI: `10.48550/arXiv.2210.13416
+    <https://doi.org/10.48550/arXiv.2210.13416>`_
+
+    """
+
+    def __init__(self, input_numb_field, output_numb_field,
+                 filter_dim, stride, model=None):
+        """Continuous Convolution 2D implementation
+
+        The algorithm expects input to be in the form:
+        $$[B \times N_{in} \times N \times D]$$
+        where $B$ is the batch_size, $N_{in}$ is the number of input
+        fields, $N$ the number of points in the mesh, $D$ the dimension
+        of the problem.
+
+        The algorithm returns a tensor of shape:
+        $$[B \times N_{out} \times N' \times D]$$
+        where $B$ is the batch_size, $N_{out}$ is the number of output
+        fields, $N'$ the number of points in the mesh, $D$ the dimension
+        of the problem.
+
+        :param input_numb_field: number of fields in the input
+        :type input_numb_field: int
+        :param output_numb_field: number of fields in the output
+        :type output_numb_field: int
+        :param filter_dim: dimension of the filter
+        :type filter_dim: tuple/ list
+        :param stride: stride for the filter
+        :type stride: dict
+        :param model: neural network for inner parametrization,
+        defaults to None
```

Comment:
wrong indentation

Reply:
solved in [a531e0e](https://github.com/mathLab/PINA/pull/69/commits/a531e0e9a2e135bae61dca44e4c6a6c18a76fb6b)